libcaf_core/caf/telemetry/metric_registry.hpp

```diff
@@ -51,18 +51,17 @@ class CAF_CORE_EXPORT metric_registry {
   /// Returns a gauge metric family. Creates the family lazily if necessary, but
   /// fails if the full name already belongs to a different family.
   /// @param prefix The prefix (namespace) this family belongs to. Usually the
-  ///               application or protocol name, e.g., `http`. The prefix `caf`
-  ///               as well as prefixes starting with an underscore are
-  ///               reserved.
+  /// application or protocol name, e.g., `http`. The prefix `caf` as well as
+  /// prefixes starting with an underscore are reserved.
   /// @param name The human-readable name of the metric, e.g., `requests`.
   /// @param labels Names for all label dimensions of the metric.
   /// @param helptext Short explanation of the metric.
   /// @param unit Unit of measurement. Please use base units such as `bytes` or
-  ///             `seconds` (prefer lowercase). The pseudo-unit `1` identifies
-  ///             dimensionless counts.
+  /// `seconds` (prefer lowercase). The pseudo-unit `1` identifies dimensionless
+  /// counts.
```

Comment:
Sorry, but you've got that wrong. The text is the documentation of `@param unit` and should be placed to the right:

```cpp
/// @param unit Detailed text ...
///             ... continue on line 2 ...
///             ... continue on line 3.
```

Reply:
Something seems off with my clang-format.. Sorry for that. I'll have a look at that